bin/github-repo-metadata

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Please revert this back to bash

Reply:
Are you sure you want to revert back to `bash`? The only "advantage" of using `bash` vs `sh` here is that with `bash`, `github-repo-metadata` will NOT run under Alpine.